aircan/dags/api_ckan_import_to_bq.py

```diff
@@ -0,0 +1,58 @@
+
+import logging
+import time
+import json
+import ast
+from datetime import date, datetime
+
+# Local imports
+from aircan.dependencies.google_cloud.bigquery_handler import bq_import_csv
+
+# Third-party library imports
+from airflow import DAG
+from airflow.exceptions import AirflowException
+
+from airflow.models import Variable
+from airflow.operators.python_operator import PythonOperator
+from airflow.utils.dates import days_ago
+
+
+args = {
+    'start_date': days_ago(0),
+    'params': { 
+        "path": "path/to/my.csv", 
+        "format": "CSV",
+        "ckan_resource_id": "res-id-123",
+        "schema": "['field1', 'field2']", 
+        "ckan_api_key": "API_KEY",
+        "ckan_site_url": "URL",
+        "output_bucket": str(date.today()),
+        "bq_project_id": "bq-project",
+        "bq_dataset_id": "bq-dataset",
+        "bq_table_name": "bq-table"
+
+    }
+}
+
+dag = DAG(
+    dag_id='ckan_api_import_to_bq',
+    default_args=args,
+    schedule_interval=None
+)
+
+def task_import_resource_to_bq(gc_file_url, table_schema, bq_project_id, bq_dataset_id, bq_table_name, **kwargs):
```

Comment:
Also looking at the suggesitons at https://github.com/datopian/aircan/pull/71, you may want to remove all the parameters this wrapper receives, leaving only `**context` i.e. `def task_import_resource_to_bq(**context): ` and recover the params from `context`